extensions/etcdsnapshot/config.go

```diff
@@ -4,13 +4,19 @@ const (
 	ConfigurationFileKey = "snapshotInput"
 )
 
+type ReplaceNodes struct {
```

Comment:
nit on the name of this. I would think `ReplaceRoles` or something of the like would be more clear? up to suggestions. Current name sounds like it would be names of nodes, or a bool on whether or not to replace the nodes instead of specifying roles to replace.

Reply:
I can update the name, that's fine with me. I'll make sure to update the README and stuff as well.

Reply:
Resolved